queries/elixir.scm

```diff
@@ -0,0 +1,60 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+
+(call
+  target: (_) @functionCallee
+) @functionCall @functionCallee.domain
+
+(keywords
+  (pair
+    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+    value: (_) @value
+  ) @collectionItem @collectionItem.trailing.start.endOf
+  (
+    ","
+  )
+?
+ @collectionItem.trailing.start.startOf
+  (#allow-multiple! @collectionItem)
+)
+
+;; %{:a => "lorem", "b" => "ipsum", 3 => "dolor"},
+(map_content
+  (_)? @_.leading.start.endOf
+  .
+  (binary_operator
+    left: (_) @collectionKey
+    right: (_) @value
+  ) @collectionItem @collectionKey.domain @value.domain @_.leading.end.startOf @_.trailing.start.endOf
```

Comment:
unfortunately you can't have more than 3 captures on a single node 😕. They get silently dropped by the tree-sitter parser 😅. Another argument for splitting `collectionItem` handling into its own pattern

Reply:
Good to know 😅